### PR TITLE
【T2】containerservice wave2

### PR DESCRIPTION
**Pull request by msyyc**

### CODE REVIEW

File: sdk/containerservice/azure-mgmt-containerservice/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,90 @@
...
+  supported options, see the [parameters accept in init documentation of azure-core](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/core/azure-core/CLIENT_LIBRARY_DEVELOPER.md#available-policies)
+- You can't import a `version` module anymore, use `__version__` instead
+- Operations that used to return a `msrest.polling.LROPoller` now returns a `azure.core.polling.LROPoller` and are prefixed with `begin_`.
+- Exceptions tree have been simplified and most exceptions are now `azure.core.exceptions.HttpResponseError` (`CloudError` has been removed).
+- Most of the operation kwarg have changed. Some of the most noticeable:
+
+  - `raw` has been removed. Equivalent feature can be found using `cls`, a callback that will give access to internal HTTP response for advanced user
+  - For a complete set of
+  supported options, see the [parameters accept in Request documentation of azure-core](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/core/azure-core/CLIENT_LIBRARY_DEVELOPER.md#available-policies)
```
Comment (changlong-liu) [inline sdk/containerservice/azure-mgmt-containerservice/CHANGELOG.md:26]: shall be in the same line with "For a complete set of" ?

Comment (msyyc) [reply sdk/containerservice/azure-mgmt-containerservice/CHANGELOG.md]: yes and changed.

File: sdk/containerservice/azure-mgmt-containerservice/azure/mgmt/containerservice/v2020_09_01/models/_models.py [hunk 2]
```diff
@@ -0,0 +1,2577 @@
...
+        self.enable_pod_security_policy = kwargs.get('enable_pod_security_policy', None)
+        self.network_profile = kwargs.get('network_profile', None)
+        self.aad_profile = kwargs.get('aad_profile', None)
+        self.auto_scaler_profile = kwargs.get('auto_scaler_profile', None)
+        self.api_server_access_profile = kwargs.get('api_server_access_profile', None)
+        self.disk_encryption_set_id = kwargs.get('disk_encryption_set_id', None)
+        self.identity_profile = kwargs.get('identity_profile', None)
+
+
+class Components1Umhcm8SchemasManagedclusteridentityPropertiesUserassignedidentitiesAdditionalproperties(msrest.serialization.Model):
```
Comment (changlong-liu) [inline sdk/containerservice/azure-mgmt-containerservice/azure/mgmt/containerservice/v2020_09_01/models/_models.py:609]: This is the same issue in https://github.com/Azure/azure-sdk-for-python/issues/14150, consider to generate again after this PR merged: https://github.com/Azure/azure-rest-api-specs/pull/11190

CC to @jsntcy

File: sdk/containerservice/azure-mgmt-containerservice/azure/mgmt/containerservice/v2020_09_01/models/_models.py [hunk 3]
```diff
@@ -0,0 +1,2577 @@
...
+        self,
+        **kwargs
+    ):
+        super(UserAssignedIdentity, self).__init__(**kwargs)
+        self.resource_id = kwargs.get('resource_id', None)
+        self.client_id = kwargs.get('client_id', None)
+        self.object_id = kwargs.get('object_id', None)
+
+
+class ComponentsQit0EtSchemasManagedclusterpropertiesPropertiesIdentityprofileAdditionalproperties(UserAssignedIdentity):
```
Comment (changlong-liu) [inline sdk/containerservice/azure-mgmt-containerservice/azure/mgmt/containerservice/v2020_09_01/models/_models.py:666]: same issue, need add more x-ms-client-name in swagger

File: sdk/containerservice/azure-mgmt-containerservice/azure/mgmt/containerservice/v2020_09_01/models/_models.py [hunk 4]
```diff
@@ -0,0 +1,2577 @@
...
+        **kwargs
+    ):
+        super(CloudErrorBody, self).__init__(**kwargs)
+        self.code = kwargs.get('code', None)
+        self.message = kwargs.get('message', None)
+        self.target = kwargs.get('target', None)
+        self.details = kwargs.get('details', None)
+
+
+class Components1Q1Og48SchemasManagedclusterAllof1(msrest.serialization.Model):
```
Comment (changlong-liu) [inline sdk/containerservice/azure-mgmt-containerservice/azure/mgmt/containerservice/v2020_09_01/models/_models.py:482]: same issue, need add more x-ms-client-name in swagger

**Pull request closed without merging**